tests/clickhouse-test

```diff
@@ -1792,6 +1792,56 @@ def reportCoverage(args):
         """
     )
 
+def reportLogPatterns(args):
+    query = """
+        WITH
```

Comment:
This function is intended to print some stats to logs in the CI. We do not have any data from previous runs in the CI, so we don't really need this condition because it will not filter anything in the CI. I added it just in case to avoid too long full scan queries when testing it locally

Reply:
Ah, that makes sense.